python-runnables/resize-node-pool/runnable.json

```diff
@@ -27,12 +27,39 @@
             "description": "Cluster (in DSS)",
             "mandatory": true
         },
+        {
```

Comment:
I think we should try to have consistent UI between creation/update. So if we cannot remove initial node pool. I would put the checkbox below number of nodes and then the hidden min/max field